makefile

```diff
@@ -1,61 +0,0 @@
-# Binaries
-UGLIFY = ./node_modules/.bin/uglifyjs
-BROWSERIFY = ./node_modules/.bin/browserify
-KARMA = ./node_modules/karma/bin/karma
-JSHINT = ./node_modules/.bin/jshint
-DEREQUIRE = ./node_modules/.bin/derequire
-
-# Build debug and minified libraries
-all: \
-	dist/tangram.min.js \
-	dist/tangram.debug.js
-
-debug: dist/tangram.debug.js
-
-dist/tangram.debug.js: .npm $(shell ./build_deps.sh)
-	node build.js --debug=true --require './src/module.js' --runtime | $(DEREQUIRE) > dist/tangram.debug.js
-
-dist/tangram.min.js: dist/tangram.debug.js
-	$(UGLIFY) dist/tangram.debug.js -c warnings=false -m -o dist/tangram.min.js
-	@gzip dist/tangram.min.js -c | wc -c | awk '{ printf "%.0fk minified+gzipped\n", $$1 / 1024 }'
```

Comment:
Here's the line to be executed, but it needs to be able to be wrapped in double quotes in the npm script, and I couldn't figure out how to escape the double quotes needed for the printf statement
```
gzip dist/tangram.min.js -c | wc -c | awk '{ printf "%.0fk minified+gzipped\n", $1 / 1024 }'
```